codebase/composer.lock

```diff
@@ -7602,13 +7836,8 @@
             "suggest": {
                 "drupal/transliterate_filenames": "Sanitizes filenames when they are uploaded so they don't break your repository."
             },
+            "default-branch": true,
             "type": "drupal-module",
-            "extra": {
-                "patches_applied": {
-                    "Routing definitions": "patches/islandora.routing.yml.patch",
```

Comment:
This made me nervous at first that the patches were not applied. When I looked around online I found a site that said patches are now listed in `vendor/composer/installed.json`, so I confirmed that all our patches are still listed there.  Sharing this so others have this context as well. https://github.com/cweagans/composer-patches/issues/338   Folks are asking for it to be put back in, and looks like it will at some point.